### DISCUSSION TITLE
Spacy with Llama-2

### DISCUSSION THREAD

**Discussion by BadrinathMJ** [explosion/spacy-llm#295] [Ideas]
Can we integrate the spacy-llm with Llama2 for named entity recognition?

**Comment by rmitsch** [comment]
Hi @BadrinathMJ, yes, every task/model combination is possible.

**Reply by BadrinathMJ** [reply]
But for fine tuning purpose we need to do the annotations of named entities extraction. If we directly pass the prompt on huge corpus of text . There may be chances of hallucinations in response by Llama 2. So should we first do the annotations and train on  those annotations and test it large corpus of text for named entity extraction?

**Reply by rmitsch** [reply]
> So should we first do the annotations and train on those annotations and test it large corpus of text for named entity extraction?

I'm not sure I follow. What exactly is your use case?

**Reply by BadrinathMJ** [reply]
@rmitsch I want extract the named entities from legal documents. e.g. There huge corpus of text documents , I want to fine tune it using LLama2 . After finetuning it, want to test it on test document for extraction of named entities.

**Reply by rmitsch** [reply]
`spacy-llm` currently doesn't support custom LLMs, only those listed in the [docs](https://spacy.io/usage/large-language-models#models). It's on our roadmap though.
